src/shared/theme.js

```diff
@@ -0,0 +1,47 @@
+const THEME = {};
+
+THEME.header = {};
+THEME.header.left = {};
+THEME.header.middle = {};
+THEME.header.right = {};
+
+THEME.header.height = "55px";
```

Comment:
i've actually been wishing for quite some time to _shrink_ the header down from 50px rather than expanding it further, since we have such limited space and are trying to do a lot of things with the pixels we have to work with. As long as we're doing this style overhaul, would you mind seeing if we can  move in that direction? playing around in devtools i'm able to get a nice looking result at 40px height, perhaps we can aim for that? it will take a bit of fiddling with buttons and stuff too... but i want those pixels! :-)